hw_diag/utilities/blockchain.py

```diff
@@ -9,13 +9,18 @@ def get_helium_blockchain_height():
     Possible exceptions:
     TypeError - if the key ['data']['height'] in response is not found.
     """
-    result = requests.get('https://api.helium.io/v1/blocks/height')
-    if result.status_code == 200:
-        result = result.json()
-        try:
-            result = result['data']['height']
-        except KeyError:
-            raise KeyError(
-                "Not found value from key ['data']['height'] in json"
-            )
-        return result
+    try:
+        result = requests.get('https://api.helium.io/v1/blocks/height',
+                              timeout=5)
+        if result.status_code == 200:
+            result = result.json()
+            try:
+                result = result['data']['height']
+            except KeyError:
+                raise KeyError(
+                    "Not found value from key ['data']['height'] in json"
+                )
+            return result
+    except (requests.exceptions.ConnectTimeout,
```

Comment:
Why bother catching these exceptions if we just rethrow them? Might as well remove the outter try/except to simplify the code in that case.

Reply:
I removed the outer try/except in the code here, I originally was handling it in blockchain.py but it was much nicer to handle it outside in tasks